### DISCUSSION TITLE
Qt5 build release and debug separately. How to combine them?

### DISCUSSION THREAD

**Discussion by Catty82** [microsoft/vcpkg#40444] [Q&A]
After running
`vcpkg install qt5`
I get its release and debug separately, debug is in /debug/bin and /debug/lib.
is there some method to combine them like as Qt officially configured them in their distros?
Note if Qt5 building from sources by manual configure and jom (nmake) then release and debug will placed into same dir.

**Comment by autoantwort** [comment]
There is currently no way to change the layout of the resulting files.
